ui/components/app/edit-gas-fee-popover/edit-gas-item/index.scss

```diff
@@ -51,6 +51,54 @@
     text-align: right;
     width: 10%;
 
+    &__container {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      flex-direction: column;
+      width: 100%;
+      height: 100%;
+
+      img {
+        margin-bottom: 8px;
+      }
+
+      &__message {
+        width: 100%;
+      }
+
+      &__dialog {
+        background-color: $Orange-500;
+        border-radius: 30px;
+        margin: 4px 0;
+        text-align: center;
+        padding: 4px;
+      }
+
+      &__label {
+        width: 50%;
+      }
+
+      &__value {
+        width: 50%;
+      }
+
+      p {
+        margin-bottom: 0 !important;
+      }
```

Comment:
It should be possible to avoid `!important` I think

Reply:
Unfortunately I'm not finding a different way to override this `margin-bottom` setting set here otherwise: https://github.com/MetaMask/metamask-extension/blob/42fea3d32329f55004d9038c93db51e130a4f4da/ui/components/ui/info-tooltip/index.scss#L56

Reply:
typography also allow passing margin, does that one helps ?

Reply:
Unfortunately it doesn't seem to... I'm actually already passing in `margin={0}` to all the typography components, which I honestly don't think is doing anything at this point. Going to remove them in fact.

Reply:
Here's what I get without the `!important`
<img width="234" alt="Screen Shot 2021-11-23 at 12 36 21 PM" src="https://user-images.githubusercontent.com/34557516/143083952-22820fd6-f79f-4e0e-9b83-9e2b214e2dff.png">

Reply:
Margin in typography does work for me, let me check again

Reply:
@jpuri Were you able to get it to work in the context of this PR?

Reply:
Yeh it does not works. I see bit of issue here, our typography components are block components, we are using those as inline here, it should rather be a typography component here wrapping inline spans / strong tags wrapped in it.